### PR TITLE
Fixed failing user visits homepage tests

### PR DESCRIPTION
**Pull request by sujan-sube**
Verified that all 9 test cases pass.

### CODE REVIEW

**Comment by arumoy-shome** [comment]
We don't have a Sign In page anymore, we are signing in from the homepage itself so we should move all that code into `user_visits_homepage_spec.rb` and delete `user_visits_signin_page_spec.rb`. Also we need to rename `sign up` to `register` everywhere.

File: spec/features/user_visits_homepage_spec.rb [hunk 1]
```diff
@@ -9,14 +9,14 @@
 
   scenario "successfully redirects to sign in page" do
     visit root_path
-    click_link 'Sign in'
+    click_button "Sign in"
```
Comment (arumoy-shome) [inline spec/features/user_visits_homepage_spec.rb:12]: Hmm, based on the redesigns that I made, this test shouldn't even be passing (this is based on the previous design). I think you are running your tests based on master, not the branch I am on. Did you pull down my branch locally and create a branch off of there?

Comment (sujan-sube) [inline spec/features/user_visits_homepage_spec.rb:12]: I did a git pull origin then made a branch off of origin/example-home-page-layout. I think when the "Sign In" button is clicked it goes to the /sessions/new path which has "Sign In" in a H1 tag, so the test passes (which shouldn't be happening).

Comment (arumoy-shome) [inline spec/features/user_visits_homepage_spec.rb:12]: I think you need to:
1. `git fetch origin/example-home-page-layout` to grab the remote branch
2. `git checkout origin/example-home-page-layout`
3. then checkout a new branch from there

Comment (arumoy-shome) [inline spec/features/user_visits_homepage_spec.rb:12]: also, when you click the `Sign In` button it hits the `SessionsController#create` action.

**Comment by arumoy-shome** [comment]
Closing this since @sujan-sube will be updating the tests in a new PR branched off of master!

**Pull request closed without merging**